docs/content/operations/including-extensions.md

```diff
@@ -3,31 +3,47 @@ layout: doc_page
 ---
 # Including Extensions
 
-Druid uses a module system that allows for the addition of extensions at runtime.
+Druid uses a module system that allows for the addition of extensions at runtime. To instruct Druid to load extensions, follow the steps below.
 
-## Specifying extensions
+## Download extensions
 
-Druid extensions can be specified in the `common.runtime.properties`. There are two ways of adding druid extensions currently.
+Core Druid extensions are already bundled in the Druid release tarball. You can get them by downloading the tarball at [druid.io](http://druid.io/).
+Unpack the tarball, you will see an ```extensions``` folder that contains all the core extensions, along with a ```hadoop-dependencies``` folder
+where it contains all the hadoop extensions. Each extension will have its own folder that contains extension jars. However, due to license issue,
+we didn't package `mysql-metadata-storage` extension in it, in order to get it, you can download it from [druid.io](http://druid.io/), unpack it and then
```

Comment:
Here you should put a period after "extension in it" and then start a new sentence.  I think there are two many "it"s jere.  How about replacing the first one ("extension in it") with 
"we didn't package the mysql-metadata-storage extension in the extensions folder".